spec/system/articles/user_edits_an_article_spec.rb

```diff
@@ -4,8 +4,15 @@
   let(:template) { file_fixture("article_published.txt").read }
   let(:user) { create(:user) }
   let(:article) { create(:article, user: user, body_markdown: template) }
+  let(:svg_image) { file_fixture("svg_image.txt").read }
 
   before do
+    SiteConfig.main_social_image = "https://i.ibb.co/NYsy5Fm/JTL-Logo.jpg"
```

Comment:
There are two image files in `spec/fixtures/files` called `800x600.png` and `800x600.jpg`, but I do not know how to reference them in a `:let` statement (like I did for the `svg-image`).
So in order to set the necessary `SiteConfig` values, I referenced some online files I hosted. But in my mind this is not good practice ❌ 
So I welcome suggestions for improvement 😄

Reply:
It looks like at least on existing spec uses dummyimage.com: spec/requests/admin/configs_spec.rb references `https://dummyimage.com/300x300`. I'd go with that!

```suggestion
    SiteConfig.main_social_image = "https://dummyimage.com/800x600.jpg"
```